arch.mk

```diff
@@ -266,7 +266,7 @@ ifeq ($(TARGET),imx_rt)
 
   ifeq ($(MCUXPRESSO_CPU),MIMXRT1052DVJ6B)
     ARCH_FLASH_OFFSET=0x60000000
-    CFLAGS+=-I$(MCUXPRESSO)/boards/evkmimxrt1050/xip/
+    CFLAGS+=-I$(MCUXPRESSO)/boards/evkbimxrt1050/xip/
```

Comment:
One last thing to note: the `EVK-MIMXRT1050` eval board was deprecated in early 2018, as it had preliminary silicon, and replaced by the `EVKB-MIMXRT1050`. The last SDK version supporting this target was v2.3.

I assume the EVK**B** is the board rev you wanted to target anyway, since `config/examples/imx-rt1050.config` points to an SDK package for the EVKB, but `arch.mk` had a path expecting the EVK board, causing some compiler errors. I fixed it by correcting this path. 

If you need to maintain support for the original EVK because of a customer requirement, let me know.